docs/book/src/getting-started/installation.md

```diff
@@ -13,7 +13,9 @@ The helm chart repository URL has changed to `https://kubernetes-sigs.github.io/
 
 #### Supported kubernetes versions
 
-Recommended Kubernetes version:
+The minimum supported Kubernetes version of Secrets Store CSI Driver is n-4 versions of the latest stable Kubernetes release per [Kubernetes Supported Versions policy](https://kubernetes.io/releases/version-skew-policy/).
+
+##### Minimum Kubernetes versions
 
 - **v1.16.0+** (For Linux)
```

Comment:
According to https://kubernetes.io/releases/patch-releases/ it seems like Kubernetes only supports 1.19 and newer currently.

Should we update accordingly? I know it _should_ work on 1.16 but may be better for us be more proactive in walking that forward as we only have e2e tests for 19, 20, 21, and 22.

Reply:
Rather than updating versions here every time, I think we should remove section entirely or just point to `supported releases` section. WDYT?

Reply:
That works for me too